content/rc/administration/setup/create-subscription.md

```diff
@@ -25,22 +27,21 @@ To create a new subscription:
 1. In the Redis Cloud menu, click **Subscriptions**.
 1. At the bottom of the page, click ![Add](/images/rs/icon_add.png#no-click "Add").
 1. Select your subscription configuration:
+
     1. Select a cloud provider: **Amazon AWS**, **Microsoft Azure**, **Google Cloud Platform**
-    1. Select the region that you want the subscription to use, for example: `us-central1`
-    1. In the Redis Cloud service levels, select the memory or throughput limit for your subscription:
-        - Essentials - You can also select either [Replication]({{< relref "/rc/concepts/clustering.md" >}}) or Multi-Availability Zone (Multi-AZ)
-        - Pro - You can also select a subscription with [Redis on Flash (RoF)]({{< relref "/rs/concepts/memory-architecture/redis-flash.md" >}}) support
+    1. Select the region that you want the subscription to use, for example: `us-central1`.
+    1. In the Redis Cloud service levels, select the memory or throughput limit for your subscription.  To learn about your options, see [Cloud Pricing](https://redislabs.com/redis-enterprise-cloud/pricing/).
+<!--
+        - Fixed - You can also select either [Replication]({{< relref "/rc/concepts/clustering.md" >}}) or Multi-Availability Zone (Multi-AZ)
+        - Flexible - You can also select a subscription with [Redis on Flash (RoF)]({{< relref "/rs/concepts/memory-architecture/redis-flash.md" >}}) support
         - Customize - You can:
             - [**Build a Plan**]({{< relref "/rc/administration/setup/customize-pro.md" >}}) to follow a sizing process where you tell us your requirements and we give you a subscription that's just right for you.
             - **Request Pricing** to customize an Ultimate subscription that includes annual payments, premium support, and customer success packages.
-1. After you select a subscription configuration:
-    1. Review the subscription configuration.
+-->
+4. After you select a subscription configuration, update technical details as requested.
 
-        For Pro subscriptions only:
-
-        1. Enable **Multi-AZ** to get improved high availability when multiple availability zones are available.
-        1. Enable **Persistent Storage Encryption** to attach encrypted EBS volumes to your instances so that the persistent storage for your subscription is encrypted.
-    1. Enter the IP range for the subscription in **Deployment CIDR**.
+    1. Review the subscription configuration.
+    1. Update technical details according to your needs, such as **Deployment CIDR** and other values as prompted.
```

Comment:
Do we have a description of what "Deployment CIDR" is anywhere? It also might be useful to define other options for the user, if they aren't familiar with the implications of each choice.

Reply:
That's a great idea, but beyond the scope of the work item.  We'll add this to a later rewrite.